src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM.java

```diff
@@ -229,7 +229,7 @@ private static String getCloneUrl(BitbucketRepository repo) {
                 .stream()
                 .filter(link -> "http".equals(link.getName()))
                 .findFirst()
-                .orElseThrow(() -> new BitbucketClientException("No HttpClone url", -404, null))
+                .orElseThrow(() -> new BitbucketClientException("No HttpClone url", HTTP_NOT_FOUND, null))
```

Comment:
I think this is not quite right. `NOT_FOUND` is not true, it is just that the response did not contain the expected response. I think we should keep a negative response code, but happy to go with `-1`